src/ConnectorEdgeModule/moduleSettings.example.json

```diff
@@ -0,0 +1,82 @@
+﻿{
+  "RootDirectory": null,
+  "KeyVaultBaseUrl": null,
+  "DisableSubscriptionRestoreService": true,
+  "EnableMessageFilter": false,
+  "ConnectorId": "###",
+  "Logging": {
+    "LogLevel": {
+      "Default": "debug",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    }
+  },
+  "OpcUa": {
+    "DefaultServerBrowseDepth": 3,
+    "NodeBrowseDepth": 1,
+    "EnableRegisteredNodes": false,
+    "SubscriptionBatchSize": 100,
+    "ReadBatchSize": 100,
+    "RegisterNodeBatchSize": 100,
+    "AwaitSessionLockTimeoutSecs": 3,
+    "ReconnectIntervalSecs": 10,
+    "NativeSettings": {
+      "Topic": "configuration",
+      "ConsumerGroup": "...",
+      "ReadBatchSize": 9999
+    }
+  },
+  "Communication": {
+    "SchemaUrl": "https://someSchemaStore.com/schemas/",
+    "Shared": {
```

Comment:
Shared between which components?
From a user perspective, the settings should be understandable, so I am just validating the arrangement and grouping according to this perspective.

Reply:
Shared between the communication sections. Also the shared section has a 'type' field. So if the the 'Shared' section under 'Communication' is of type kafka then all other Kafka sections (Telemetry, Command & Response) can inherit their config from the Shared section. But each lower section can also override the shared section's config. This is to prevent bloating of the configuration file. Especially if you a;; kafka endpoints are on the same kafka cluster/server.

Reply:
That makes sense now.